docs/interfaces/index.md

```diff
@@ -23,6 +23,7 @@ Our Fusion interfaces are
 <tr markdown><td markdown>[embedR](https://github.com/KxSystems/embedR)</td><td markdown>[Call **R** from q](r.md)</td></tr>
 <tr markdown><td markdown>[ffi](https://github.com/KxSystems/ffi)</td><td markdown>[Call **C/C++** from q](using-c-functions.md)</td></tr>
 <tr markdown><td markdown>[javakdb](https://github.com/KxSystems/javakdb)</td><td markdown>**Java** client for kdb+ [:fontawesome-regular-map:](../wp/java-api/index.md "White paper: Java API for kdb+")</td>
+<tr markdown><td markdown>[pykx](https://code.kx.com/pykx)</td><td markdown>Integrate **Python** and q code</td></tr>
```

Comment:
If we're going to have two lines in this table that say "Integrate Python and q code" can you say a few words about the difference between the options? Or, should the pyq line be deleted?

Reply:
Had pinged ones responsible earlier for what they'd prefer here. Will update when they get back to me.

Reply:
I suggest remove PyQ as a suggested interface